### ISSUE TITLE
Select bulk items and apply 'Keep origin translation'

### ISSUE THREAD

**Issue by khoipro** [GlotPress/GlotPress-WP#1663]
Hi,

There are many times I saw the string should not be translated, such as date format, and city names. Is there any way we could bulk select items and apply 'Keep origin translation' to avoid single approval action?

Thanks,

**Comment by amieiro** [comment]
If the string should not be translated, the best approach is to don't make it translatable. 
We have recently added some error checks to GlotPress in https://github.com/GlotPress/GlotPress/pull/1644 and at translate.w.org we are going to add some checks for the WordPress core project in https://github.com/WordPress/wordpress.org/pull/162: timezone strings, start of week number, unexpected slug format or unexpected timezone date format, so if you think some translations should have a specific format, the best approach is to add an error check in GlotPress or at translate.w.og.

**Comment by khoipro** [comment]
The discussion in Slack here: https://wordpress.slack.com/archives/C02RP50LK/p1690356179365669

We hope GTE (who can approve bulk items) uses it to avoid extra time for unused strings. If we keep it no translation, the percentage for releasing will be stuck.

**Comment by marcarmengou** [comment]
I leave this here in case you think it might be a possible solution to this: https://meta.trac.wordpress.org/ticket/7132

**Comment by pedro-mendonca** [comment]
There are cases where the string might not have a translation in a particular language. Some examples that make sense might be some city names, country names, in Portuguese there are some that already have a translation but the vast majority coule easily be bulk copied from original. Another example is English variants, probably for English UK many things can just be copied from original.

**Comment by pedro-mendonca** [comment]
@MarcArmengou that is a different situation.

I think this is about the need of a new bulk action: "Copy original"

**Comment by nilovelez** [comment]
Yes, [GotDict browser extension](https://chrome.google.com/webstore/detail/glotdict/jfdkihdmokdigeobcmnjmgigcgckljgl) adds the "Copy from original" bulk action and it's really useful for GTEs/CLPTEs to get untranslatable strings out of the way and prevent that volunteers waste time on them.
